spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -3563,6 +3565,10 @@ def setPlainText(self, txt):
     def focusOutEvent(self, event):
         """Reimplement Qt method"""
         self.sig_focus_changed.emit()
+        self.clear_extra_selections('ctrl_click')
+        QApplication.restoreOverrideCursor()
+        self.__cursor_changed = False
+        self._last_hover_uri = None

```

Comment:
Now that I think about it, could we refactor these actions? we are basically doing the same on both methods and what we are doing is kinda like a `reset editor stuff`

Thoughts @jnsebgosselin ?

Reply:
Yes, I agree with this. Do you have a name in mind for the method that I could use?

Reply:
Also @goanpeca, change your review to `request changes` so that Carlos don't ninja merge this before I make the changes :P

Reply:
hahaha will do 

Well, it should begin with an underscore and we could call it

```python
def _restore_editor_cursor_and_selections(self):
```

Or somethinng similar ;-), up to you

Reply:
The only thing I am not sure is if it is correct to add `self._last_hover_uri = None` in there.